larrecodnn/ImagePatternAlgs/Modules/CheckCNNScore_module.cc

```diff
@@ -60,15 +61,17 @@ class pdsp::CheckCNNScore : public art::EDAnalyzer {
   std::vector<short> wire;
   std::vector<double> charge;
   std::vector<double> peakt;
-  std::vector<double> score_inel;
-  std::vector<double> score_el;
-  std::vector<double> score_none;
+  std::vector<double> score_0;
+  std::vector<double> score_1;
+  std::vector<double> score_2;
+  std::vector<double> score_3;
```

Comment:
```suggestion
  std::array<std::vector<double>, 4> scores;
```
Then see usage below.